.husky/pre-commit

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+. "$(dirname "$0")/_/husky.sh"
+
+yarn lint && yarn build
```

Comment:
Let's set up GitHub Actions for this and rely on GitHub Checks on each PR then rather than local hooks? More reliable and also good for Dependabot PRs so we see if they pass.

Reply:
It's only for developers.
just to be sure all your stuff is working,